filebeat/docs/modules.asciidoc

```diff
@@ -0,0 +1,182 @@
+[[filebeat-modules]]
+== Modules
+
+beta[]
+
+Filebeat modules simplify the collection, parsing, and visualization of common
+log formats.
+
+A typical module (say, for the Nginx logs) is composed of one ore
+more filesets (in the case of Nginx, `access` and `error`). A fileset contains
+the following:
+
+* Filebeat prospector configurations, which contain the default paths were to
+  look or the log files. These default paths depend on the operating system.
+  The Filebeat configuration is also responsible with stitching together
+  multiline events when needed.
+
+* Elasticsearch {elasticsearch}/ingest.html[Ingest Node] pipeline definition,
+  which is used to parse the log lines.
+
+* Fields definitions, which are used to configure Elasticsearch with the
+  correct types for each field. They also contain short descriptions for each
+  of the fields.
+
+* Sample Kibana dashboards, which can be used to visualize the log files.
+
+Filebeat automatically adjusts these configurations based on your environment
+and loads them to the respective Elastic stack components.
+
+NOTE: At the moment, Filebeat modules require using the Elasticsearch
+{elasticsearch}/ingest.html[Ingest Node]. In the future, Filebeat Modules will
+be able to also configure Logstash as a more powerful alternative to Ingest
```

Comment:
I hope that by the time we release this, we'll be able to say that we have sample Logstash configs which we can mention instead: https://github.com/elastic/logstash/issues/6542

Until then, and in case that doesn't happen on time, I think it's important to let people know that we do plan to allow for migrating from Ingest Node to LS.